{'dataset_name': 'ogbg-ppa', 'seed': 0, 'checkpoint_dir': './checkpoint', 'num_workers': 4, 'cov_weight': 0.01, 'feature': 'full', 'hyperparams': {'batch_size': 32, 'epochs': 801, 'learning_rate': 0.001, 'step_size': 30, 'decay_rate': 0.8}, 'architecture': {'layers': 4, 'hidden': 256, 'pooling': 'add', 'JK': 'cat', 'nonlinear_conv': 'penalty', 'dropout': 0.5, 'exps': 1, 'leak': 0.2, 'aggr': 'mean'}, 'commit_id': 'f0c0b2f8b20a85622aa393591ca4534a6e47a7f4', 'time_stamp': '1612582245', 'directory': '../../../nlg_results/dc2-Ymq2/ogbg/board/'}
Epoch 1 training...
Evaluating...
Train: 0.04884910485933504 Validation: 0.04350332594235033 Test: 0.04793103448275862 Train loss: 16.73163577714255 Train cov: 5.370504265135907 lr: 0.001
Saving model as f0c0b2f_1612582245_ogbg-ppa_1.tar... Traceback (most recent call last):
  File "./main.py", line 242, in <module>
    main()
  File "./main.py", line 225, in main
    os.path.join(config.checkpoint_dir, filename))
  File "/opt/conda/lib/python3.6/site-packages/torch/serialization.py", line 369, in save
    with _open_file_like(f, 'wb') as opened_file:
  File "/opt/conda/lib/python3.6/site-packages/torch/serialization.py", line 234, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/opt/conda/lib/python3.6/site-packages/torch/serialization.py", line 215, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: './checkpoint/f0c0b2f_1612582245_ogbg-ppa_1.tar'
